client/performance-profiler/components/metrics-insight/insight-content.tsx

```diff
@@ -98,10 +100,23 @@ export const InsightContent: React.FC< InsightContentProps > = ( props ) => {
 		);
 	};
 
+	const renderLoadingMessage = () => {
+		if ( isWpscanLoading ) {
+			return (
+				<LLMMessage
+					message={ translate(
+						"We're still checking some details of your site to make the best possible recommendations."
+					) }
```

Comment:
I would add the `rotate` here 😄 , because when the delay is long, it seems it is stuck as there is no impression of progress

Reply:
That's a good point, I added the `rotate` prop in 38aa57aeb63

Reply:
Thank you!